app/src/test/java/com/duckduckgo/app/trackerdetection/TrackerDetectorClientTypeTest.kt

```diff
@@ -27,15 +29,18 @@ import org.junit.Assert.assertEquals
 import org.junit.Assert.assertNull
 import org.junit.Before
 import org.junit.Test
+import org.junit.runner.RunWith
 
+@RunWith(AndroidJUnit4::class)
```

Comment:
There are quite a few changes exactly like this one to use the AndroidJUnit4 runner instead of Robolectric.